Packs/CortexXDR/IncidentFields/incidentfield-XDR_Alert_Search_Results.json

```diff
@@ -0,0 +1,164 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "xdralertsearchresults",
```

Comment:
Wouldn't XDR Search Alerts Results be more accurate?

Reply:
"XDR Search Alerts Results" implies that i'm searching for results that related to alert. what i'm doing is searching for alerts.